pkg/store/store.go

```diff
@@ -131,7 +132,9 @@ type UserStore interface {
 	RetrieveUser(ctx context.Context, emailOrUserID any) (*models.User, error)
 	UpdateUser(context.Context, *models.User) error
 	SetUserPassword(ctx context.Context, userID ulid.ULID, password string) error
+	SetUserLastLogin(ctx context.Context, userID ulid.ULID, lastLogin time.Time) error
```

Comment:
Added this so the user's last login timestamp wouldn't get nulled out on update.